events-api/src/database/migrations/1727368986894-create-tickets.sql

```diff
@@ -0,0 +1,14 @@
+-- UP
+CREATE TABLE IF NOT EXISTS tickets (
+    ticket_id TEXT PRIMARY KEY,
+    ticket_quantity INT NOT NULL,
+    ticket_price DECIMAL NOT NULL,
+    event_id TEXT NOT NULL,
+    ticket_created_at DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    ticket_updated_at DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+
+    FOREIGN KEY (event_id) REFERENCES events(event_id)
```

Comment:
What should happen if event was deleted?

Reply:
I think I have committed `DROP TABLE IF EXISTS tickets` in another commit...

Reply:
> What should happen if event was deleted?
I think I have committed DROP TABLE IF EXISTS tickets in another commit...

Reply:
What will happen with ticket when even is deleted from the table? It will point to non-existing event, which can lead to issues